.github/workflows/ci.yml

```diff
@@ -240,14 +240,23 @@ jobs:
             librealm-android-armeabi-v7a
             librealm-android-arm64-v8a
 
-# Linux jobs
+# Dart jobs
 
-  tests-linux:
-    runs-on: ubuntu-latest
-    name: Tests Linux
+  tests-dart:
+    runs-on: ${{ matrix.platform == 'linux' && 'ubuntu' || matrix.platform }}-${{ matrix.architecture || 'latest' }}
+    name: Tests ${{ matrix.platform }} ${{ matrix.architecture }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - platform: macos
+          - platform: macos
```

Comment:
hos is this playing with artifacts in binary/macos

Reply:
Those are each running on a separate machine, so there should be nothing to overwrite.